DATA/UNIVERSE/SYSTEMS/Br04/Br04.ini

```diff
@@ -3774,6 +3774,335 @@ faction = fc_g_grp, 0.33
 faction = fc_m_grp, 0.33
 faction = fc_ou_grp, 0.33
 
+[Zone]

```

Comment:
Ideally patrol paths come after most object and zone defintions, but before vignettes and random background noise pops